data/json/harvest.json

```diff
@@ -237,6 +237,14 @@
     "type": "harvest",
     "entries": [ { "drop": "cherries", "base_num": [ 6, 18 ], "scale_num": [ 0, 0.5 ] } ]
   },
+  {
+    "id": "coffee_harv",
+    "type": "harvest",
+    "entries": [
+      { "drop": "coffee_pod", "base_num": [ 4, 10 ], "scale_num": [ 0, 0.5 ] },
+      { "drop": "tea_raw", "base_num": [ 1, 3 ], "scale_num": [ 0, 0.5 ] }
```

Comment:
Coffee tree will also drop tea?

Reply:
This is the original JSON data before #42616.

Reply:
Might I suggest the error was not that it was dropping cherries, but that it wasn't called a cherry tree?

Reply:
There is a cherry tree in `data/json/furniture_and_terrain/terrain-flora.json`

Reply:
I still think that's an error in harvesting tea from coffee tree.

Reply:
Coffee trees do not grow in New England, BTW.

Reply:
After some `git blame` tracing, I found that leaves were added in #24800.

Reply:
The link given in the PR is for either "Kentucky coffeetree" (apparently also known as "Kent coffeetree" - it is native to North America, so I'm using the Kentucky name), which does not have leaves that produce tea (but does have fruit that can be dried and roasted to make an imitation coffee - _without caffeine_, though), or for an _indoor_ planting of actual coffee trees/bushes. Is this supposed to be outdoors or indoors? There are only one or two _Coffea_ species that can take freezing for even a short period of time, and none are cultivated in New England outdoors as far as I am aware.

Reply:
WTH, yea this is WRONG.
Just because it gets used for a vaguely tea-like purpose doesn't mean it's ok for it to drop actual tea leaves, if you want to do that you need to fill out the whole lineage of "not actually tea" and "not actually coffee" items.
And confirming what @actual-nh said, the "Kentucky Coffee Tree" is NOT COFFEE.